### PR TITLE
Bug 1163732 - [Contacts] Losing connection while importing contacts f…

### PR DESCRIPTION
**Pull request by ferjm**
…rom Gmail or Outlook will show the user a screen with 'Confirmation title' and an 'Action 1' button. r=borjasalguero

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1eccea22d4ed38fba4c0c5fd5a68011bfcea3bc9

File: apps/communications/contacts/index.html [hunk 1]
```diff
@@ -102,6 +102,7 @@
     <link rel="localization" href="/shared/locales/date/date.{locale}.properties">
     <link rel="localization" href="/shared/locales/phone_types/phone_types.{locale}.properties">
     <link rel="localization" href="/shared/locales/telephony/telephony.{locale}.properties">
+    <link rel="localization" href="/shared/locales/facebook/facebook.en-US.properties">
```
Comment (borjasalguero) [inline apps/communications/contacts/index.html]: Should we add `/shared/locales/facebook/facebook.{locale}.properties`?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=451a8279003f36b485ecd4b9e7e91d9a2dd20a60

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=08f9edd014518879caf2cc9bb107190422163684

**Pull request merged**